packages/autoloader/src/autoload.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+// autoload_wordpress.php generated by automattic/autoloader
+
+$class_map = require_once dirname( __FILE__ ) . '/composer/autoload_classmap_package.php';
+
+if ( ! function_exists( 'jetpack_enqueue_library' ) ) {
+
+	global $jetpack_packages;
```

Comment:
I'm concerned we're exposing the packages and can allow any plugin to mess with our autoloading. Can we do anything to conceal it?

Reply:
That is partially the point. Another plugin should be able to make it so that it loads the latest required package.

Reply:
it all lives inside the same PHP VM, so in theory there's nothing we can do to stop another plugin from messing with our autoloading.